blocks/library/gallery/index.js

```diff
@@ -101,6 +101,7 @@ registerBlockType( 'core/gallery', {
 									buttonProps={ {
 										className: 'components-icon-button components-toolbar__control',
 										'aria-label': __( 'Edit Gallery' ),
+										tabIndex: -1,
```

Comment:
With these changes, what is the implications on block authors for managing tab index? How do we document this? Can we avoid it?

Reply:
Is it possible to hijack the tab keypress to avoid the default tabIndex behavior when in the tabbable cycle, so that we don't need buttons to be assigned with `tabIndex="-1"`?